ui_framework/src/components/context_menu/context_menu_item.js

```diff
@@ -22,7 +22,7 @@ export class KuiContextMenuItem extends Component {
       hasPanel,
       icon,
       buttonRef,
-      ...rest,
+      rest,
```

Comment:
this looks suspicious, removing the ...?

Reply:
Oh, good catch, I intended to remove the comma but must have gone crazy for a second 🦀

Reply:
https://github.com/elastic/kibana/pull/14862/commits/37c0be90efae6e48e48df73c79ca5698925245ea